pkgs/development/libraries/aws-sdk-cpp/default.nix

```diff
@@ -127,5 +120,7 @@ stdenv.mkDerivation rec {
     license = licenses.asl20;
     platforms = platforms.unix;
     maintainers = with maintainers; [ eelco orivej ];
+    # building ec2 runs out of memory: cc1plus: out of memory allocating 33554372 bytes after a total of 74424320 bytes
+    broken = ((builtins.elem "ec2" apis) || (builtins.elem "*" apis));
```

Comment:
I think that's only an issue if `stdenv.buildPlatform.is32bit`.

Reply:
At least Hydra still succeeds: https://hydra.nixos.org/job/nixpkgs/trunk/aws-sdk-cpp.x86_64-linux

Reply:
Amended.

Reply:
Yes, right. Totally missed this.